tokio-console/Cargo.toml

```diff
@@ -34,6 +34,7 @@ tonic = { version = "0.7", features = ["transport"] }
 futures = "0.3"
 tui = { version = "0.16.0", default-features = false, features = ["crossterm"] }
 tracing = "0.1"
+tracing-causality = { path = "../../tracing-causality" }
```

Comment:
can this be changed to a Git dependency rather than a path dependency, at least? i'd really like to be able to actually try out this branch without having to check that out.